metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-extlib",
-  "version": "5.3.1-rc0",
+  "version": "6.0.0-rc0",
```

Comment:
please update this to 6.0.0 (we've a rake check that prevents releases for -rc versions)

Reply:
done